### PR TITLE
fix(scss): Use consistent spacing instead of tabs

### PR DESCRIPTION
**Pull request by CuriousLearner**

### CODE REVIEW

**Comment by sabderemane** [comment]
Hi @CuriousLearner, I would like to suggest a change related to that, because I'm using EditorConfig to be sure I have the good configuration where there is one defined in a project and the current configuration sounds not adapted to the need according the changes :
```
# .editorconfig

[*.{scss,js}]
indent_style = tab
indent_size = tab
```
It should be `space` instead :)

**Comment by CuriousLearner** [comment]
Hey @sabderemane 

Yes, that sounds correct. However, now I saw other SCSS files in the repo and they all are mixed of spaces and tabs. I'll get them fixed to have consistent spaces if @django/djangoproject-com-maintainters are okay with that. If you have any other preference, I'm all ears!

**Review by camilonova — APPROVED** [review]

**Comment by pauloxnet** [comment]
> Hey @sabderemane
> 
> Yes, that sounds correct. However, now I saw other SCSS files in the repo and they all are mixed of spaces and tabs. I'll get them fixed to have consistent spaces if @django/djangoproject-com-maintainters are okay with that. If you have any other preference, I'm all ears!

I'm ok with that

**Review by emarhin — APPROVED** [review]
Not bad

**Review by emarhin — APPROVED** [review]

File: djangoproject/scss/_normalize.scss [hunk 1]
```diff
@@ -61,9 +61,12 @@ audio:not([controls]) {
 */
 
 html {
-    font-family: sans-serif; /* 1 */
-    -webkit-text-size-adjust: 100%; /* 2 */
-    -ms-text-size-adjust: 100%; /* 2 */
+    font-family: sans-serif;
+    /* 1 */
```
[the remark this answers is not in the document]
Comment (CuriousLearner) [reply djangoproject/scss/_normalize.scss:65]: Sure, I can revert them. Ideally I just place multiline comment on new lines, but this is fine.

**Comment by CuriousLearner** [comment]
This is ready for a review :)

**Comment by sabderemane** [comment]
@CuriousLearner do you think a rebase is needed ? I don't know if other files have changed since your last changes

**Pull request merged**